libvast/src/type.cpp

```diff
@@ -704,6 +704,29 @@ bool compatible(const data& lhs, relational_operator op, const type& rhs) {
   return compatible(rhs, flip(op), lhs);
 }
 
+bool is_superset(const type& x, const type& y) {
+  auto sub = caf::get_if<record_type>(&x);
+  auto super = caf::get_if<record_type>(&y);
```

Comment:
```suggestion
  auto sup = caf::get_if<record_type>(&y);
```

For consistency.

Reply:
Not feeling this one, nobody ever said supset.